participant/.env.example

```diff
@@ -7,3 +7,8 @@ DATABASE_URL=postgresql://postgres:incredible_local_secret_phrase@127.0.0.1:5432
 POSTGRES_PASSWORD=incredible_local_secret_phrase
 POSTGRES_USER=postgres
 PUBLIC_DEMO_MODE=false
+S3_ENDPOINT_URL=http://127.0.0.1:9000
+MAX_UPLOAD_SIZE_BYTES=5242880
+MAX_UPLOAD_FILECOUNT=5
+AWS_ACCESS_KEY_ID=e2e_minio
+AWS_SECRET_ACCESS_KEY=incredible_local_secret_phrase
```

Comment:
It looks like we will need to provision a IAM user with permissions to access the S3 bucket. Is that correct? 

If so, I have tf WIP for that saved for that that I'll use in a [new ticket](https://wicmtdp.atlassian.net/browse/PRP-278) I've created for making sure we get all the correct env vars to the participant ECS task